docs/4.3/kubernetes_ssh.md

```diff
@@ -159,6 +165,10 @@ spec:
         - root
       # list of Kubernetes groups this Github team is allowed to connect to
       kubernetes_groups: ["system:masters"]
+      # Optional: If not set users will imperonsate themseleves. 
+      # impersonate a kubernetes user with IAM prefix
```

Comment:
I agree the IAM prefix doesn't make a lot of sense here - if we're going to use it, we should explain that it needs to map to whatever string is being sent over by your identity provider.

Reply:
I've another ticket to review Kubernetes users, so I'll rewrite that section afterwards.